data/sdo-howto-examples.txt

```diff
@@ -92,98 +92,342 @@ PRE-MARKUP:
-        <span itemprop="afterMedia"><img itemprop="image" href="car-raised.jpg" /></span>
-      </div>
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToTip">
-        <div itemprop="text">It doesn't need to be too high.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Remove the hubcap and loosen the lug nuts.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <span itemprop="duringMedia"><img itemprop="image" href="replacing-tire.mp4" /></span>
-        <div itemprop="text">Remove the flat tire and put the spare tire on the exposed lug bolts.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Tighten the lug nuts by hand.</div>
-      </div>
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToTip">
-        <div itemprop="text">Don't use the wrench just yet.</div>
-      </div>
-    </div>
-  </div>
-  <div itemprop="step" itemscope itemtype="http://schema.org/HowToSection">
-    <div itemprop="name">Finishing up</div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Lower the jack and tighten the lug nuts with the wrench.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Replace the hubcap.</div>
-      </div>
-    </div>
-    <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToStep">
-      <div itemprop="itemListElement" itemscope itemtype="http://schema.org/HowToDirection">
-        <div itemprop="text">Put the equipment and the flat tire away.</div>
-      </div>
+    <strong><span itemprop="name">Change a Flat Tire</span></strong>
+    <div>About <span itemprop="estimatedCost" itemscope itemtype="http://schema.org/MonetaryAmount">
+        <meta itemprop="currency" content="USD"></meta>
+        <meta itemprop="value" content="20"></meta>
+    </span>$20</div>
+    <div>About <span itemprop="totalTime" content="PT30M">30 minutes</span></div>
+    <div>Necessary Items:</div>
+    <div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Spare tire</div>
```

Comment:
I believe the name of the HowToTool will not be set with this syntax. The previous way works, or if you remove the itemscope & itemtype.